### PR TITLE
lsf support

### PR DESCRIPTION
**Pull request by bpanneton**
Support for lsf

### CODE REVIEW

**Comment by chu11** [comment]
In principle looks good.  A few things

A) Could you rebase off master.  I totally screwed up with the last release.  The submission convert names need to be changed (e.g. from msubtorque to msubtorquepdsh).  I've made the fix in master, but I need you to adjust accordingly.  Totally my bad, sorry.  There will be adjustments in the script-templates directory as a result (lsf -> lsf-mpirun everywhere).

B) Could you add the appropriate information to the README file.  Basically everywhere it talks about how to submit a job via msub, add the lsf equivalent.  Every where it talks about "you'll find msub + slurm scripts in submission-scripts/....", add the appropriate lsf stuff.  There might be a few more places too, like the general header in README & README.md that says LSF is supported.

**Comment by chu11** [comment]
Oh, and update the .travis.yml file for the lsf build

**Comment by bpanneton** [comment]
This should be good and was testing with the testall

File: magpie-submission-convert [hunk 1]
```diff
@@ -79,3 +79,26 @@ then
 	export MAGPIE_TIMELIMIT_MINUTES=$(expr ${PBS_WALLTIME} / 60)
     fi
 fi
+
+if [ "${MAGPIE_SUBMISSION_TYPE}" == "lsf" ] || [ "${MAGPIE_SUBMISSION_TYPE}" == "lsfmpirun" ]
```
Comment (chu11) [inline magpie-submission-convert]: no need to check for "lsf" here, since there's no need for backwards compatibility.  We only support "lsfmpirun"

**Comment by chu11** [comment]
Really close.  Only 1 minor nit.

Also, could you copy over the minor change in the README to README.md.

**Comment by chu11** [comment]
Oh, and on the first part of the "Basic Idea" text in README & README.md, could you mention LSF.  Thanks

**Comment by chu11** [comment]
LGTM

**Pull request merged**